### DISCUSSION TITLE
Launch options?

### DISCUSSION THREAD

**Discussion by theperspectiv** [JuneStepp/OneLauncher#67] [Q&A] [answered]
Is there a way to pass arguments to the lotroclient executable when using OneLauncher? LOTRO runs best on Linux with Gamescope for me, which I usually do by putting "gamescope -w 1920 -h 1200 -b -- %command%" in the Steam launch options. Is this possible with OL?

**Comment by JuneStepp** [comment, marked as answer]
You may be able to just start OneLauncher with Gamescope. I have added this to my roadmap, though.
